Proyecto Nextflix

Carga de datos:

In [19]:
import pandas as pd

df_originals = pd.read_csv('C:/Users/Daniella Guerra/Desktop/Python - Pandas/Netflix_Project/datos/netflix_originals.csv')
df_titles = pd.read_csv('C:/Users/Daniella Guerra/Desktop/Python - Pandas/Netflix_Project/datos/netflix_titles.csv')


In [ ]:
#Visualizamos la estadistica para cada columna, asi podremos detectar donde basarnos para nuestro analisis

print(df_originals.describe())
print(df_titles.describe())



       Unnamed: 0     Runtime  IMDB Score
count  513.000000  513.000000  513.000000
mean   280.984405   94.674464    6.210916
std    166.445537   26.648121    0.968850
min      0.000000    4.000000    2.500000
25%    137.000000   87.000000    5.600000
50%    277.000000   97.000000    6.300000
75%    422.000000  108.000000    6.900000
max    583.000000  209.000000    9.000000
        Unnamed: 0  release_year
count  8807.000000   8807.000000
mean   4403.000000   2014.180198
std    2542.506244      8.819312
min       0.000000   1925.000000
25%    2201.500000   2013.000000
50%    4403.000000   2017.000000
75%    6604.500000   2019.000000
max    8806.000000   2021.000000


In [ ]:
#Identificamos valores nulos para poder trabajar en ellos

print(df_originals.isnull().sum())
print(df_titles.isnull().sum())

#Podemos ver que tenemos valores nulos, pudiendo deberse a la falta de informacion en estos caso

Unnamed: 0    0
Title         0
Genre         0
Premiere      0
Runtime       0
IMDB Score    0
Language      0
dtype: int64
Unnamed: 0         0
show_id            0
type               0
title              0
director        2634
cast             825
country          831
date_added        10
release_year       0
rating             4
duration        4813
listed_in          0
description        0
dtype: int64


In [ ]:
#Identificamos valores duplicados, donde, por suerte, no tenemos ninguno para este estudio

print(df_originals.duplicated().sum())
print(df_titles.duplicated().sum())


0
0


Analisis de columnas y estructura

In [ ]:
#Tamaño de filas y columnas

print("Tamaño de Netflix Originals:")
print(df_originals.shape)

print("\nTamaño de Netflix Titles:")
print(df_titles.shape)


Tamaño de Netflix Originals:
(513, 7)

Tamaño de Netflix Titles:
(8807, 13)


In [ ]:
#Analisis de estadisticas descriptivas
print("\nDescripción de Netflix Originals:")
print(df_originals.describe(include='all'))

print("\nDescripción de Netflix Titles:")
print(df_titles.describe(include='all'))



Descripción de Netflix Originals:
        Unnamed: 0                                     Title        Genre  \
count   513.000000                                       513          513   
unique         NaN                                       513          106   
top            NaN  David Attenborough: A Life on Our Planet  Documentary   
freq           NaN                                         1          132   
mean    280.984405                                       NaN          NaN   
std     166.445537                                       NaN          NaN   
min       0.000000                                       NaN          NaN   
25%     137.000000                                       NaN          NaN   
50%     277.000000                                       NaN          NaN   
75%     422.000000                                       NaN          NaN   
max     583.000000                                       NaN          NaN   

                Premiere     Runtime  IM

In [ ]:
#Conteo de titulos únicos para cada conjunto de datos

print("\nTítulos únicos en Netflix Originals:")
print(df_originals['Title'].nunique())

print("\nTítulos únicos en Netflix Titles:")
print(df_titles['title'].nunique())



Títulos únicos en Netflix Originals:
513

Títulos únicos en Netflix Titles:
8807


In [ ]:
#Conteo de duplicados para cada conjunto de datos

print("\nDuplicados en Netflix Originals:")
print(df_originals.duplicated().sum())

print("\nDuplicados en Netflix Titles:")
print(df_titles.duplicated().sum())



Duplicados en Netflix Originals:
0

Duplicados en Netflix Titles:
0


In [ ]:
#Conteo de valores nulos para cada conjunto de datos
print("\nValores nulos en Netflix Originals:")
print(df_originals.isnull().sum())

print("\nValores nulos en Netflix Titles:")
print(df_titles.isnull().sum())



Valores nulos en Netflix Originals:
Unnamed: 0    0
Title         0
Genre         0
Premiere      0
Runtime       0
IMDB Score    0
Language      0
dtype: int64

Valores nulos en Netflix Titles:
Unnamed: 0         0
show_id            0
type               0
title              0
director        2634
cast             825
country          831
date_added        10
release_year       0
rating             4
duration        4813
listed_in          0
description        0
dtype: int64


Tendriamos dos caso: columnas con Vlores Nulos Significativos

director: 2634 valores nulos.
duration: 4813 valores nulos (esta es una de las columnas más afectadas)
cast: 825 valores nulos.
country: 831 valores nulos.

Y columnas con Pocos Valores Nulos:

date_added: 10 valores nulos.
rating: 4 valores nulos.

En caso de estos valores, al no tener duplicados, no podemos saber ciertos valores como el director, duracion, cast o pais. Por lo que procederemos a rellenar como desconocido.

In [ ]:
merged_df['director'] = merged_df['director'].fillna('Desconocido')
merged_df['cast'] = merged_df['cast'].fillna('Cast Desconocido')
merged_df['country'] = merged_df['country'].fillna('País Desconocido')
merged_df['duration'] = merged_df['duration'].fillna('Duración Desconocida')
merged_df['date_added'] = merged_df['date_added'].fillna('Fecha Desconocida')
merged_df['rating'] = merged_df['rating'].fillna('Sin Clasificar')

In [40]:
#Verificando si ya no hay valores nulos
print("Valores nulos después del tratamiento:")
print(df_titles.isnull().sum())

Valores nulos después del tratamiento:
Unnamed: 0         0
show_id            0
type               0
title              0
director        2634
cast             825
country          831
date_added        10
release_year       0
rating             4
duration        4813
listed_in          0
description        0
dtype: int64


In [ ]:
#Calculo de generos en los conjuntos de datos

print("\nGéneros en Netflix Originals:")
print(df_originals['Genre'].value_counts())

print("\nGéneros en Netflix Titles:")
print(df_titles['listed_in'].value_counts())



Géneros en Netflix Originals:
Genre
Documentary                             132
Drama                                    73
Comedy                                   42
Romantic comedy                          35
Thriller                                 33
                                       ... 
Action-adventure                          1
Concert Film                              1
Animation / Science Fiction               1
Anthology/Dark comedy                     1
Animation/Christmas/Comedy/Adventure      1
Name: count, Length: 106, dtype: int64

Géneros en Netflix Titles:
listed_in
Dramas, International Movies                        362
Documentaries                                       359
Stand-Up Comedy                                     334
Comedies, Dramas, International Movies              274
Dramas, Independent Movies, International Movies    252
                                                   ... 
Action & Adventure, Cult Movies                       1
Action & A

In [ ]:
#Países en Netflix Originals (columna Language)
country_distribution_originals = df_originals['Language'].value_counts()
print("Distribución de idiomas en Netflix Originals:")
print(country_distribution_originals)

#Países en Netflix Titles
country_distribution_titles = df_titles['country'].value_counts()
print("\nDistribución de países en Netflix Titles:")
print(country_distribution_titles)


Distribución de idiomas en Netflix Originals:
Language
English                       352
Hindi                          28
Spanish                        26
French                         18
Italian                        14
Portuguese                      9
Indonesian                      9
Japanese                        5
English/Spanish                 5
Korean                          5
Turkish                         4
German                          4
Marathi                         3
Polish                          3
English/Japanese                2
Dutch                           2
Filipino                        2
Thai                            2
English/Hindi                   2
Norwegian                       1
Malay                           1
Swedish                         1
Spanish/Basque                  1
English/Swedish                 1
Spanish/Catalan                 1
English/Taiwanese/Mandarin      1
Thia/English                    1
Bengali                    

In [ ]:
#Idiomas en Netflix Originals
language_distribution_originals = df_originals['Language'].value_counts()
print("\nIdiomas en Netflix Originals:")
print(language_distribution_originals)

#Idiomas en Netflix Titles
language_distribution_titles = df_titles['Language'].value_counts()
print("\nIdiomas en Netflix Titles:")
print(language_distribution_titles)



Idiomas en Netflix Originals:
Language
English                       352
Hindi                          28
Spanish                        26
French                         18
Italian                        14
Portuguese                      9
Indonesian                      9
Japanese                        5
English/Spanish                 5
Korean                          5
Turkish                         4
German                          4
Marathi                         3
Polish                          3
English/Japanese                2
Dutch                           2
Filipino                        2
Thai                            2
English/Hindi                   2
Norwegian                       1
Malay                           1
Swedish                         1
Spanish/Basque                  1
English/Swedish                 1
Spanish/Catalan                 1
English/Taiwanese/Mandarin      1
Thia/English                    1
Bengali                         1
Khmer/En

KeyError: 'Language'

In [ ]:
#generos por país en Netflix Titles

genres_by_country = df_titles.groupby('country')['listed_in'].value_counts().head(10)
print("\nDistribución de géneros por país en Netflix Titles:")
print(genres_by_country)



Distribución de géneros por país en Netflix Titles:
country            listed_in                                                        
, France, Algeria  Dramas, Independent Movies, International Movies                     1
, South Korea      International TV Shows, TV Dramas                                    1
Argentina          Stand-Up Comedy                                                      8
                   Crime TV Shows, International TV Shows, Spanish-Language TV Shows    6
                   Dramas, International Movies                                         6
                   Documentaries, International Movies, Sports Movies                   4
                   Comedies, Cult Movies, International Movies                          2
                   Documentaries, International Movies                                  2
                   Dramas, Independent Movies, International Movies                     2
                   Dramas, International Movies, Thr

Netflix Titles refleja una estrategia global al ofrecer una amplia diversidad de géneros que combinan elementos internacionales, mientras adapta contenido para satisfacer audiencias locales clave, como lo demuestra la fuerte presencia de géneros como Spanish-Language TV Shows y Stand-Up Comedy en Argentina. Además que la inclusión de géneros como documentales, películas deportivas y contenido de culto mantiene el esfuerzo de Netflix por captar audiencias específicas. Este enfoque mixto, que combina globalización y regionalización permite a Netflix fortalecer su atractivo tanto en mercados internacionales como en comunidades locales

Unión de Datos

In [ ]:
#Realizar la unión
merged_df = pd.merge(df_titles, df_originals, left_on='title', right_on='Title', how='left')

print(merged_df.head())


   Unnamed: 0_x show_id     type                  title         director  \
0             0      s1    Movie   Dick Johnson Is Dead  Kirsten Johnson   
1             1      s2  TV Show          Blood & Water      Desconocido   
2             2      s3  TV Show              Ganglands  Julien Leclercq   
3             3      s4  TV Show  Jailbirds New Orleans      Desconocido   
4             4      s5  TV Show           Kota Factory      Desconocido   

                                                cast          country  \
0                                   Cast Desconocido    United States   
1  Ama Qamata, Khosi Ngema, Gail Mabalane, Thaban...     South Africa   
2  Sami Bouajila, Tracy Gotoas, Samuel Jouy, Nabi...  Unknown Country   
3                                   Cast Desconocido  Unknown Country   
4  Mayur More, Jitendra Kumar, Ranjan Raj, Alam K...            India   

           date_added  release_year rating          duration  \
0  September 25, 2021          2020  PG-

In [41]:
#Conteo de producciones originales y no-originales
originals_count = merged_df['Title'].notnull().sum()
acquired_count = merged_df['Title'].isnull().sum()

print(f"Producciones originales en Netflix Titles: {originals_count}")
print(f"Títulos no-originales en Netflix Titles: {acquired_count}")


Producciones originales en Netflix Titles: 513
Títulos no-originales en Netflix Titles: 8294


In [ ]:
total_titles = originals_count + acquired_count
percentage_originals = (originals_count / total_titles) * 100

print(f"Porcentaje de producciones originales: {percentage_originals:.2f}%")

Porcentaje de producciones originales: 5.83%


Este porcentaje bajo refleja que las producciones originales son selectivas, probablemente orientadas a diferenciar la marca Netflix y atraer audiencias específicas.
Aunque representan una pequeña parte del catálogo, suelen recibir un mayor enfoque en marketing y promoción
El 94.17% restante del catálogo corresponde a contenido adquirido.
Esto muestra que Netflix complementa sus producciones originales con adquisiciones para ofrecer un catálogo amplio y diverso que pueda satisfacer diferentes nichos de audiencia

In [42]:
# Puntuación IMDB promedio de originales
originals_imdb_mean = merged_df[merged_df['Title'].notnull()]['IMDB Score'].mean()

# Puntuación IMDB promedio de no-originales
acquired_imdb_mean = merged_df[merged_df['Title'].isnull()]['IMDB Score'].mean()

print(f"Puntuación IMDB promedio de producciones originales: {originals_imdb_mean:.2f}")
print(f"Puntuación IMDB promedio de títulos no-originales: {acquired_imdb_mean:.2f}")


Puntuación IMDB promedio de producciones originales: 6.21
Puntuación IMDB promedio de títulos no-originales: nan


La calificación promedio IMDB de las producciones originales de Netflix es 6.21, lo que indica una recepción moderada por parte de los espectadores, sugiriendo que aunque las producciones originales son estratégicamente seleccionadas, no destacan como consistentemente sobresalientes en términos de calidad percibida. Por otro lado, la falta de datos para calcular la puntuación promedio de los títulos no originales (NaN) sugiere que este grupo carece de calificaciones en la base de datos utilizada, lo que limita una comparación directa entre producciones originales y adquiridas. Esto podría deberse a que las calificaciones de IMDB no se recopilan para todo el catálogo adquirido, o que estas producciones no tienen suficiente visibilidad para recibir puntuaciones. Para hacer este calculo necesitariamos datos de la columna IMDB score que, en este caso no lo tienen los titulos no-originales.

In [70]:
# Puntuación IMDB promedio por país en producciones originales
imdb_by_country = merged_df[merged_df['Title'].notnull()].groupby('country')['IMDB Score'].mean()

print("\nPuntuación IMDB promedio por país:")
print(imdb_by_country)



Puntuación IMDB promedio por país:
country
Argentina                                 6.420000
Australia, United States                  7.600000
Austria                                   6.850000
Belgium, Netherlands                      7.100000
Belgium, United Kingdom, United States    5.300000
                                            ...   
United States, United Kingdom             8.200000
United States, United Kingdom, Canada     6.700000
United States, United Kingdom, Germany    6.600000
United States, United Kingdom, India      7.800000
Unknown Country                           6.421429
Name: IMDB Score, Length: 82, dtype: float64


Las coproducciones entre Estados Unidos y otros países, especialmente el Reino Unido y la India, tienden a recibir puntuaciones IMDB más altas, lo que podría deberse a mayores presupuestos o alcance global.
Los países con puntuaciones más bajas, como Belgium, United Kingdom, United States (5.3), podrían reflejar experimentos menos exitosos en términos de recepción.
Netflix parece tener un impacto global significativo, con representaciones variadas en las puntuaciones.


In [71]:
# Títulos originales mejor calificados
top_originals = merged_df[merged_df['Title'].notnull()].sort_values(by='IMDB Score', ascending=False).head(10)

print("\nProducciones originales mejor calificadas:")
print(top_originals[['title', 'IMDB Score', 'Genre', 'country']])


Producciones originales mejor calificadas:
                                            title  IMDB Score  \
1886     David Attenborough: A Life on Our Planet         9.0   
4293                      Springsteen on Broadway         8.5   
5893  Winter on Fire: Ukraine's Fight for Freedom         8.4   
3386                       Dancing with the Birds         8.3   
5152                       Cuba and the Cameraman         8.3   
3273                                        Klaus         8.2   
1857        The Three Deaths of Marisela Escobedo         8.2   
1171                                   Seaspiracy         8.2   
2097                               Rising Phoenix         8.1   
2038                           My Octopus Teacher         8.1   

                                     Genre  \
1886                           Documentary   
4293                          One-man show   
5893                           Documentary   
3386                           Documentary   
5152      

Netflix ha logrado producir contenido original de alta calidad, especialmente en el género documental, que domina las producciones mejor calificadas.
La diversidad geográfica y de temáticas sugiere una estrategia intencionada para abarcar tanto el mercado global como temas de relevancia universal.
Este éxito en producciones originales fortalece la marca Netflix como una plataforma que prioriza la calidad en lugar de la cantidad en sus contenidos originales.

Uso de loc_iloc

In [6]:
import pandas as pd

df_originals = pd.read_csv('C:/Users/Daniella Guerra/Desktop/Python - Pandas/Netflix_Project/datos/netflix_originals.csv')
df_titles = pd.read_csv('C:/Users/Daniella Guerra/Desktop/Python - Pandas/Netflix_Project/datos/netflix_titles.csv')

# Realizar la unión
merged_df = pd.merge(df_titles, df_originals, left_on='title', right_on='Title', how='left')

# Verificar el resultado
print(merged_df.head())

   Unnamed: 0_x show_id     type                  title         director  \
0             0      s1    Movie   Dick Johnson Is Dead  Kirsten Johnson   
1             1      s2  TV Show          Blood & Water              NaN   
2             2      s3  TV Show              Ganglands  Julien Leclercq   
3             3      s4  TV Show  Jailbirds New Orleans              NaN   
4             4      s5  TV Show           Kota Factory              NaN   

                                                cast        country  \
0                                                NaN  United States   
1  Ama Qamata, Khosi Ngema, Gail Mabalane, Thaban...   South Africa   
2  Sami Bouajila, Tracy Gotoas, Samuel Jouy, Nabi...            NaN   
3                                                NaN            NaN   
4  Mayur More, Jitendra Kumar, Ranjan Raj, Alam K...          India   

           date_added  release_year rating   duration  \
0  September 25, 2021          2020  PG-13     90 min   
1 

In [ ]:

movies = merged_df.loc[merged_df['type'] == 'Movie']
print(movies)


      Unnamed: 0_x show_id   type                             title  \
0                0      s1  Movie              Dick Johnson Is Dead   
6                6      s7  Movie  My Little Pony: A New Generation   
7                7      s8  Movie                           Sankofa   
9                9     s10  Movie                      The Starling   
12              12     s13  Movie                      Je Suis Karl   
...            ...     ...    ...                               ...   
8801          8801   s8802  Movie                           Zinzana   
8802          8802   s8803  Movie                            Zodiac   
8804          8804   s8805  Movie                        Zombieland   
8805          8805   s8806  Movie                              Zoom   
8806          8806   s8807  Movie                            Zubaan   

                           director  \
0                   Kirsten Johnson   
6     Robert Cullen, José Luis Ucha   
7                      Haile G

In [8]:
tv_shows = merged_df.loc[merged_df['type'] == 'TV Show', ['title', 'duration']]
print(tv_shows)


                      title   duration
1             Blood & Water  2 Seasons
2                 Ganglands        NaN
3     Jailbirds New Orleans        NaN
4              Kota Factory  2 Seasons
5             Midnight Mass        NaN
...                     ...        ...
8795        Yu-Gi-Oh! Arc-V  2 Seasons
8796             Yunus Emre  2 Seasons
8797              Zak Storm        NaN
8800     Zindagi Gulzar Hai        NaN
8803            Zombie Dumb  2 Seasons

[2676 rows x 2 columns]


In [9]:
movies_with_director = merged_df.loc[merged_df['type'] == 'Movie', ['title', 'director']]
print(movies_with_director)


                                 title                       director
0                 Dick Johnson Is Dead                Kirsten Johnson
6     My Little Pony: A New Generation  Robert Cullen, José Luis Ucha
7                              Sankofa                   Haile Gerima
9                         The Starling                 Theodore Melfi
12                        Je Suis Karl            Christian Schwochow
...                                ...                            ...
8801                           Zinzana                Majid Al Ansari
8802                            Zodiac                  David Fincher
8804                        Zombieland                Ruben Fleischer
8805                              Zoom                   Peter Hewitt
8806                            Zubaan                    Mozez Singh

[6131 rows x 2 columns]


In [10]:
titles_2018 = merged_df.loc[merged_df['release_year'] == 2018, ['title', 'listed_in']]
print(titles_2018)


                                                  title  \
21                               Resurrection: Ertugrul   
37                                          Angry Birds   
73                                         King of Boys   
94                                            Show Dogs   
96    If I Leave Here Tomorrow: A Film About Lynyrd ...   
...                                                 ...   
8732                                      White Chamber   
8743                                           Wildlife   
8774                                          يوم الدين   
8775                                    Yeh Meri Family   
8803                                        Zombie Dumb   

                                              listed_in  
21    International TV Shows, TV Action & Adventure,...  
37                                Kids' TV, TV Comedies  
73                         Dramas, International Movies  
94                   Children & Family Movies, Comedies  
9

In [11]:
no_director = merged_df.loc[merged_df['director'].isna(), ['title']]
print(no_director)


                                    title
1                           Blood & Water
3                   Jailbirds New Orleans
4                            Kota Factory
10    Vendetta: Truth, Lies and The Mafia
14        Crime Stories: India Detectives
...                                   ...
8795                      Yu-Gi-Oh! Arc-V
8796                           Yunus Emre
8797                            Zak Storm
8800                   Zindagi Gulzar Hai
8803                          Zombie Dumb

[2634 rows x 1 columns]


Filtrado  con iloc

In [12]:
first_5 = merged_df.iloc[:5, [2, 4, 6]]  # Ajusta los índices según la posición de las columnas
print(first_5)


      type         director        country
0    Movie  Kirsten Johnson  United States
1  TV Show              NaN   South Africa
2  TV Show  Julien Leclercq            NaN
3  TV Show              NaN            NaN
4  TV Show              NaN          India


In [13]:
last_5 = merged_df.iloc[-5:, [2, 11]]  # Ajusta los índices según la posición de las columnas
print(last_5)


         type                                       listed_in
8802    Movie                  Cult Movies, Dramas, Thrillers
8803  TV Show          Kids' TV, Korean TV Shows, TV Comedies
8804    Movie                         Comedies, Horror Movies
8805    Movie              Children & Family Movies, Comedies
8806    Movie  Dramas, International Movies, Music & Musicals


In [14]:
first_10_columns_2_to_5 = merged_df.iloc[:10, 1:5]
print(first_10_columns_2_to_5)


  show_id     type                             title  \
0      s1    Movie              Dick Johnson Is Dead   
1      s2  TV Show                     Blood & Water   
2      s3  TV Show                         Ganglands   
3      s4  TV Show             Jailbirds New Orleans   
4      s5  TV Show                      Kota Factory   
5      s6  TV Show                     Midnight Mass   
6      s7    Movie  My Little Pony: A New Generation   
7      s8    Movie                           Sankofa   
8      s9  TV Show     The Great British Baking Show   
9     s10    Movie                      The Starling   

                        director  
0                Kirsten Johnson  
1                            NaN  
2                Julien Leclercq  
3                            NaN  
4                            NaN  
5                  Mike Flanagan  
6  Robert Cullen, José Luis Ucha  
7                   Haile Gerima  
8                Andy Devonshire  
9                 Theodore Melfi 

In [15]:
last_7_columns = merged_df.iloc[-7:, [2, 4, 6]]  # Ajusta los índices según la posición de las columnas
print(last_7_columns)


         type         director                       country
8800  TV Show              NaN                      Pakistan
8801    Movie  Majid Al Ansari  United Arab Emirates, Jordan
8802    Movie    David Fincher                 United States
8803  TV Show              NaN                           NaN
8804    Movie  Ruben Fleischer                 United States
8805    Movie     Peter Hewitt                 United States
8806    Movie      Mozez Singh                         India


In [16]:
rows_15_to_25_columns_1_to_4 = merged_df.iloc[15:26, :4]
print(rows_15_to_25_columns_1_to_4)


    Unnamed: 0_x show_id     type  \
15            15     s16  TV Show   
16            16     s17    Movie   
17            17     s18  TV Show   
18            18     s19    Movie   
19            19     s20  TV Show   
20            20     s21  TV Show   
21            21     s22  TV Show   
22            22     s23    Movie   
23            23     s24    Movie   
24            24     s25    Movie   
25            25     s26  TV Show   

                                                title  
15                                  Dear White People  
16  Europe's Most Dangerous Man: Otto Skorzeny in ...  
17                                    Falsa identidad  
18                                          Intrusion  
19                                             Jaguar  
20    Monsters Inside: The 24 Faces of Billy Milligan  
21                             Resurrection: Ertugrul  
22                                    Avvai Shanmughi  
23       Go! Go! Cory Carson: Chrissy Takes the Whe

In [17]:
rows_15_to_25_columns_1_to_4 = merged_df.iloc[15:26, :4]
print(rows_15_to_25_columns_1_to_4)


    Unnamed: 0_x show_id     type  \
15            15     s16  TV Show   
16            16     s17    Movie   
17            17     s18  TV Show   
18            18     s19    Movie   
19            19     s20  TV Show   
20            20     s21  TV Show   
21            21     s22  TV Show   
22            22     s23    Movie   
23            23     s24    Movie   
24            24     s25    Movie   
25            25     s26  TV Show   

                                                title  
15                                  Dear White People  
16  Europe's Most Dangerous Man: Otto Skorzeny in ...  
17                                    Falsa identidad  
18                                          Intrusion  
19                                             Jaguar  
20    Monsters Inside: The 24 Faces of Billy Milligan  
21                             Resurrection: Ertugrul  
22                                    Avvai Shanmughi  
23       Go! Go! Cory Carson: Chrissy Takes the Whe